scalyr_agent/util.py

```diff
@@ -220,6 +220,9 @@ def rfc3339_to_datetime( string ):
     # split the string in to main time and fractional component
     parts = string.split(".")
 
+    if parts[0].endswith( 'Z' ):
```

Comment:
Add a comment to describe what this is doing.

Reply:
Will do.